SWIPs/swip-message_to_honey.md

```diff
@@ -13,28 +13,33 @@ Created: 31-07-2019
 
 ## Simple summary
 <!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the SWIP.-->
-Nodes in the SWARM network can send various types of messages. Currently, the SWARM developers have limited data available to set the optimal price of these messages relative to each other and we expect to adjust the relative prices of messages based on advancing insights. It is important that an update of such prices happens simultaneously on all nodes in the network or otherwise accounting imbalances between nodes will come into existence, potentially severely harming the connectivity of the network. 
-An on-chain price oracle, managed by a multi-signature wallet of SWARM stakeholders provides a clean way for updating relative prices; the SWARM source code will never require an update but instead references the address of the oracle which updates its quoted prices at predictable times.
+For the purpose of this SWIP, price, if not explicitly noted differently, refers to the price in `honey`, Swarm's internal accounting unit. It is not a currency with exchange value, but a unit of reference which allows peers to account for the services exchanged. It can be best understood as accounting for the amount of bytes exchanged.
+
+Nodes in the Swarm network can send various types of messages. Every message type represents different resource usage in the network. In order for the incentives system to be optimally tuned, the current assumption is that every message type will require a different price to account for the resources used.
+
+Currently, the Swarm developers have no empirical data nor theoretical references available to determine the optimal price of these messages relative to each other. We expect extensive experimentation and as a result will have to progressively adjust the relative prices of messages. It is important that an update of the prices happens simultaneously on all nodes in the network or otherwise accounting imbalances between nodes will arise, potentially severely harming the connectivity of the network. 
+
+An on-chain price oracle, managed by a multi-signature wallet of Swarm stakeholders, provides a clean way for updating relative prices; the Swarm source code will never require an update but instead references the address of the oracle which updates its quoted prices at predictable times.
```

Comment:
Here maybe also the issue of governance arises. Transparency of the change process? How do we notify users/ the network?

Reply:
It indeed arises here too but to a lesser degree: we can compare it to how op-codes are priced in Ethereum. Also here, the op-codes are changed by developers (see: [discussion](https://ethereum-magicians.org/t/opcode-repricing/3024)), based on observed behavior of the network. 
For us, it will be more difficult for two reasons: 
* We don't have metrics yet on which we want to base a change of *relative* prices. Even if we would have metrics, we don't know yet how to gather such metrics from all nodes - apart from sampling.
* In Ethereum, prices are changed by deveopers, but node operators (full nodes/miners) are the ones who install the software, thereby vetting the change. The proposed mechanism does not have such a check.